### ISSUE TITLE
[Bug]: fix the error

### ISSUE THREAD

**Issue by onlynewbie** [mangiucugna/json_repair#89]
### Version of the library

0.33

### Describe the bug

There are redundant fields of ''' in the front and back in the middle, and there is a } missing in the middle, but the last json has been fixed to have a problem

### How to reproduce

'''json[{"Material_itself": "Perovskite-based solar cells", "Material_processing": {"Synthesis_method": "", "Steps": ["Nanostructured Al2O3 were employed as a mesoporous scaffold for fabrication of such solar cells."], "Substrate": "Al2O3", "Reaction_time": "", "Reaction_temperature": "", "Pressure": "", "Raw_materials": "", "Reaction_conditions": "", "Growth_template": "", "Growth_orientation": "", "Characterization_method": "Random walk numerical simulation based on hopping model", "Processing_conclusion": "our simulation."}, "Material_structure": {"Material_dimension": "", "Structure": "Morphology and thickness of scaffold", "Structural_composition": "", "Growth_orientation": "", "Numerical_indicator": "", "Evaluative_indicator": ""}, "Material_properties": {"Property": "unique properties", "Numerical_indicator": "", "Evaluative_indicator": ""},'''' {"Material_itself": "Al2O3", "Material_processing": {"Synthesis_method": "", "Steps": ["Nanostructured Al2O3 were employed as a mesoporous scaffold for fabrication of such solar cells."], "Substrate": "", "Reaction_time": "", "Reaction_temperature": "", "Pressure": "", "Raw_materials": "", "Reaction_conditions": "", "Growth_template": "", "Growth_orientation": "", "Characterization_method": "", "Processing_conclusion": ""}, "Material_structure": {"Material_dimension": "", "Structure": "scaffold", "Structural_composition": "", "Growth_orientation": "", "Numerical_indicator": "", "Evaluative_indicator": ""}, "Material_properties": {"Property": "very important for optimization of conversion efficiency of the cells", "Numerical_indicator": "", "Evaluative_indicator": ""}}]'''

This is the result：
[{"Material_itself": "Perovskite-based solar cells", "Material_processing": {"Synthesis_method": "", "Steps": ["Nanostructured Al2O3 were employed as a mesoporous scaffold for fabrication of such solar cells."], "Substrate": "Al2O3", "Reaction_time": "", "Reaction_temperature": "", "Pressure": "", "Raw_materials": "", "Reaction_conditions": "", "Growth_template": "", "Growth_orientation": "", "Characterization_method": "Random walk numerical simulation based on hopping model", "Processing_conclusion": "our simulation."}, "Material_structure": {"Material_dimension": "", "Structure": "Morphology and thickness of scaffold", "Structural_composition": "", "Growth_orientation": "", "Numerical_indicator": "", "Evaluative_indicator": ""}, "Material_properties": {"Property": "unique properties", "Numerical_indicator": "", "Evaluative_indicator": ""}, " {\"Material_itself\": \"Al2O3\", \"Material_processing\": {\"Synthesis_method\": \"\", \"Steps\": [\"Nanostructured Al2O3 were employed as a mesoporous scaffold for fabrication of such solar cells.\"], \"Substrate\": \"\", \"Reaction_time\": \"\", \"Reaction_temperature\": \"\", \"Pressure\": \"\", \"Raw_materials\": \"\", \"Reaction_conditions\": \"\", \"Growth_template\": \"\", \"Growth_orientation\": \"\", \"Characterization_method\": \"\", \"Processing_conclusion\": \"\"}, \"Material_structure\": {\"Material_dimension\": \"\", \"Structure\": \"scaffold\", \"Structural_composition\": \"\", \"Growth_orientation\": \"\", \"Numerical_indicator\": \"\", \"Evaluative_indicator\": \"\"}, \"Material_properties\": {\"Property\": \"very important for optimization of conversion efficiency of the cells\", \"Numerical_indicator\": \"\", \"Evaluative_indicator\": \"\"}}]": ""}]

### Expected behavior

hope fix，thank you

**Comment by onlynewbie** [comment]
Hello, I tried the new 0.34 package again but the result is still wrong

This is the txt：
'''json[{"Material_itself": "Perovskite-based solar cells", "Material_processing": {"Synthesis_method": "", "Steps": ["Nanostructured Al2O3 were employed as a mesoporous scaffold for fabrication of such solar cells."], "Substrate": "Al2O3", "Reaction_time": "", "Reaction_temperature": "", "Pressure": "", "Raw_materials": "", "Reaction_conditions": "", "Growth_template": "", "Growth_orientation": "", "Characterization_method": "Random walk numerical simulation based on hopping model", "Processing_conclusion": "our simulation."}, "Material_structure": {"Material_dimension": "", "Structure": "Morphology and thickness of scaffold", "Structural_composition": "", "Growth_orientation": "", "Numerical_indicator": "", "Evaluative_indicator": ""}, "Material_properties": {"Property": "unique properties", "Numerical_indicator": "", "Evaluative_indicator": ""},'''' {"Material_itself": "Al2O3", "Material_processing": {"Synthesis_method": "", "Steps": ["Nanostructured Al2O3 were employed as a mesoporous scaffold for fabrication of such solar cells."], "Substrate": "", "Reaction_time": "", "Reaction_temperature": "", "Pressure": "", "Raw_materials": "", "Reaction_conditions": "", "Growth_template": "", "Growth_orientation": "", "Characterization_method": "", "Processing_conclusion": ""}, "Material_structure": {"Material_dimension": "", "Structure": "scaffold", "Structural_composition": "", "Growth_orientation": "", "Numerical_indicator": "", "Evaluative_indicator": ""}, "Material_properties": {"Property": "very important for optimization of conversion efficiency of the cells", "Numerical_indicator": "", "Evaluative_indicator": ""}}]'''"""

The fixed result：
[{"Material_itself": "Perovskite-based solar cells", "Material_processing": {"Synthesis_method": "", "Steps": ["Nanostructured Al2O3 were employed as a mesoporous scaffold for fabrication of such solar cells."], "Substrate": "Al2O3", "Reaction_time": "", "Reaction_temperature": "", "Pressure": "", "Raw_materials": "", "Reaction_conditions": "", "Growth_template": "", "Growth_orientation": "", "Characterization_method": "Random walk numerical simulation based on hopping model", "Processing_conclusion": "our simulation."}, "Material_structure": {"Material_dimension": "", "Structure": "Morphology and thickness of scaffold", "Structural_composition": "", "Growth_orientation": "", "Numerical_indicator": "", "Evaluative_indicator": ""}, "Material_properties": {"Property": "unique properties", "Numerical_indicator": "", "Evaluative_indicator": ""}, " {\"Material_itself\": \"Al2O3\", \"Material_processing\": {\"Synthesis_method\": \"\", \"Steps\": [\"Nanostructured Al2O3 were employed as a mesoporous scaffold for fabrication of such solar cells.\"], \"Substrate\": \"\", \"Reaction_time\": \"\", \"Reaction_temperature\": \"\", \"Pressure\": \"\", \"Raw_materials\": \"\", \"Reaction_conditions\": \"\", \"Growth_template\": \"\", \"Growth_orientation\": \"\", \"Characterization_method\": \"\", \"Processing_conclusion\": \"\"}, \"Material_structure\": {\"Material_dimension\": \"\", \"Structure\": \"scaffold\", \"Structural_composition\": \"\", \"Growth_orientation\": \"\", \"Numerical_indicator\": \"\", \"Evaluative_indicator\": \"\"}, \"Material_properties\": {\"Property\": \"very important for optimization of conversion efficiency of the cells\", \"Numerical_indicator\": \"\", \"Evaluative_indicator\": \"\"}}]": ""}]

But the Correct result：
[
    {
        "Material_itself": "Perovskite-based solar cells",
        "Material_processing": {
            "Synthesis_method": "",
            "Steps": [
                "Nanostructured Al2O3 were employed as a mesoporous scaffold for fabrication of such solar cells."
            ],
            "Substrate": "Al2O3",
            "Reaction_time": "",
            "Reaction_temperature": "",
            "Pressure": "",
            "Raw_materials": "",
            "Reaction_conditions": "",
            "Growth_template": "",
            "Growth_orientation": "",
            "Characterization_method": "Random walk numerical simulation based on hopping model",
            "Processing_conclusion": "our simulation."
        },
        "Material_structure": {
            "Material_dimension": "",
            "Structure": "Morphology and thickness of scaffold",
            "Structural_composition": "",
            "Growth_orientation": "",
            "Numerical_indicator": "",
            "Evaluative_indicator": ""
        },
        "Material_properties": {
            "Property": "unique properties",
            "Numerical_indicator": "",
            "Evaluative_indicator": ""
        }
    },
    {
        "Material_itself": "Al2O3",
        "Material_processing": {
            "Synthesis_method": "",
            "Steps": [
                "Nanostructured Al2O3 were employed as a mesoporous scaffold for fabrication of such solar cells."
            ],
            "Substrate": "",
            "Reaction_time": "",
            "Reaction_temperature": "",
            "Pressure": "",
            "Raw_materials": "",
            "Reaction_conditions": "",
            "Growth_template": "",
            "Growth_orientation": "",
            "Characterization_method": "",
            "Processing_conclusion": ""
        },
        "Material_structure": {
            "Material_dimension": "",
            "Structure": "scaffold",
            "Structural_composition": "",
            "Growth_orientation": "",
            "Numerical_indicator": "",
            "Evaluative_indicator": ""
        },
        "Material_properties": {
            "Property": "very important for optimization of conversion efficiency of the cells",
            "Numerical_indicator": "",
            "Evaluative_indicator": ""
        }
    }
]

**Comment by mangiucugna** [comment]
I see why, when I tested I removed all trailing quotes at the end. In this case there's nothing I can do about it. The json is repaired in a way that looks weird but makes sense: the quotes at the end close a string and that is considered a totally valid object key and then an empty object value is added.

If you remove the trailing quotes by yourself you will see that is repaired correctly. Unfortunately this output is extremely ambiguous.
